### ISSUE TITLE
Unable to set title for DialogTitle, etc. after updating from pre.478 to pre.616 

### ISSUE THREAD

**Issue by arkivanov** [JetBrains/kotlin-wrappers#2095]
The following code produces an error in the marked lines.

```
'inline fun String?.unaryPlus(): Unit' can't be called in this context by implicit receiver. Use the explicit one if necessary
``` 

```kotlin
    Dialog {
        open = true
        onClose = { _, _ -> }
        ariaLabelledBy = "dialog-title"
        ariaDescribedBy = "dialog-message"

        DialogTitle {
            +"title" // Error here
        }

        DialogContent {
            DialogContentText {
                +"message" // Error here
            }
        }

        DialogActions {
            Button {
                onClick = {}
                +"Close" // This works fine
            }
        }
    }
```

**Comment by turansky** [comment]
Temp WA - use version `pre.608`

**Comment by arkivanov** [comment]
Thanks! Version `pre.608` works!

**Comment by aerialist7** [comment]
Will be ready in `pre.618`

**Comment by arkivanov** [comment]
Thanks for the quick fix!

**Comment by AngeloFilaseta** [comment]
Using this closed issue since `pre.609` also broke some components of mine created using a custom adapter. At that time I remember using this page to create them: [Using Packages from npm](https://kotlinlang.org/docs/using-packages-from-npm.html).

I had the same problem of OP on those components:
```
'inline fun String?.unaryPlus(): Unit' can't be called in this context by implicit receiver. Use the explicit one if necessary
```
I also was unable to import other components inside them:
```
'inline operator fun <P : Props> ElementType<Props>.invoke(): Unit' can't be called in this context by implicit receiver. Use the explicit one if necessary
```

So, I tried to change all the `Props`  to `PropsWithChildren` and now everything works fine.  Working Code example follows:
```kotlin

@file:JsModule("react-bootstrap/Navbar")
@file:JsNonModule

@JsName("default")
external val Navbar: ComponentClass<NavbarProps>

external interface NavbarProps : PropsWithChildren {
    var bg: String
    var variant: String
}
```
Is my solution ok?
How should we treat external components obtained  now?
I tried checking the [pre.609 changelog](https://github.com/JetBrains/kotlin-wrappers/compare/pre.608...pre.609) but I'm unsure of how Props' behaviour changed in this specific scenario.
Thank you and sorry for using a closed issue. I could move this into a new one if necessary.
